src/main.js

```diff
@@ -6,6 +6,7 @@ require('prototype_creep_move');
 require('prototype_roomPosition');
 require('prototype_room_init');
 require('prototype_room_costmatrix');
+
 require('visualizer');
 require('screepsplus');
 require('visualizer');
```

Comment:
:-) I was not necessary meant as a question, more like a: 'Would you be so kind to remove one of the two?'

Thx, for fixing.